assets/page.html

```diff
@@ -160,10 +77,9 @@ <h2 class="accordion-header" id="setting_accordion_heading">
             <br>{EXTRA_FOOTER}
             <br>Made using the <a href="https://bootswatch.com/darkly/">Darkly Bootstrap Theme</a> by Thomas Park, which was released under the <a href="https://github.com/thomaspark/bootswatch/blob/95df99d76147797cbcb1014b639805add2327f65/LICENSE">MIT License</a>.
             <br>
-            <br><div style="width:256px;height:512px;"></div>
```

Comment:
Why was this removed? This was a very intentional bugfix.

Reply:
Since the images have a width and height attribute now, the page is not "jumping" anymore